src/main/src/main/java/org/geoserver/config/impl/ContactInfoImpl.java

```diff
@@ -50,6 +82,8 @@ public void setId(String id) {
 
     @Override
     public String getAddress() {
+        if (internationalAddress != null)
```

Comment:
It seems the logic is different from the FeatureTypeInfoImpl case, e.g.:

```java
        if (this.internationalTitle == null || this.internationalTitle.toString().equals("")) {
            return new GrowableInternationalString(title);
        }
        return this.internationalTitle;
```

I'd recommend a consistent approach unless there is a good reason to do otherwise. Since this bloc is repeated so much, it could also be moved to a utility class/method receiving the base string and the international string.

Reply:
fixed